### PR TITLE
QPPSF-7546: Add PI measure validation for specific PI measures

### PR DESCRIPTION
**Pull request by saquino0827**
### Information
- https://jira.cms.gov/browse/QPPSF-7546

### Changes proposed in this PR:
- Adds measure validation for PI_HIE_1, PI_HIE_4, and both their respective exclusion measures to be rejected if submitted with PI_HIE_5/

### Checklist 
- [X] All JUnit tests pass (`mvn clean verify`).
- [X] New unit tests written to cover new functionality.
- [X] Added and updated JavaDocs for non-test classes and methods.
- [X] No local design debt. Do you feel that something is "ugly" after your changes?
- [X] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

**Review by KyleApfel — APPROVED** [review]
Good refactoring and clean up here. Thanks Sam!

File: converter/src/test/java/gov/cms/qpp/conversion/validate/PiSectionValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+import gov.cms.qpp.conversion.model.Node;
+import gov.cms.qpp.conversion.model.TemplateId;
+import gov.cms.qpp.conversion.model.error.Detail;
+import gov.cms.qpp.conversion.model.error.ProblemCode;
+import gov.cms.qpp.conversion.model.error.correspondence.DetailsErrorEquals;
+
+class PiSectionValidatorTest {
+
+	private static final String VALID_ACI_MEASURE = "ACI_EP_1";
```
Comment (shadford) [inline converter/src/test/java/gov/cms/qpp/conversion/validate/PiSectionValidatorTest.java:19]: Does this check for the 2017 ACI_EP_1 measure only or does it need to switch between PI_EP_1?

**Review by shadford — APPROVED** [review]
Looks Good!

File: converter/src/test/java/gov/cms/qpp/conversion/validate/PiSectionValidatorTest.java (hunk 1 above)
Comment (saquino0827) [reply converter/src/test/java/gov/cms/qpp/conversion/validate/PiSectionValidatorTest.java:19]: This is an old test that I was refactoring but it uses old test data. Good catch, as the actual measure name is just test data here but it should be updated as well.

**Review by shadford — APPROVED** [review]

**Pull request merged**